largest-series-product/largest_series_product_test.go

```diff
@@ -56,6 +66,9 @@ const pe1k = "73167176531330624919225119674426574742355349194934" +
 	"71636269561882670428252483600823257530420752963450"
 
 func TestLargestSeriesProduct(t *testing.T) {
+	if TestVersion != testVersion {
+		t.Fatalf("Found TestVersion = %v, want %v", TestVersion, testVersion)
+	}
```

Comment:
People will most likely not notice these things. It's only after this bites you a few times that you become wary of int sizes.

I'd prefer things to fail during compilation, in line with the principle of failing as early as possible.